package chapter.six.two.three.one

import org.junit.jupiter.api.Test

internal class ThrowableExceptionTest {
    @Test
    fun maintest() {
        ThrowableException.main(arrayOf())
    }
}